src/api/showScores.js

```diff
@@ -0,0 +1,14 @@
+const listScore = document.querySelector('.list-scores');
+
+function showScores(gameScores) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer `arrow functions ` over normal functions, so as to keep your _function_ syntax simpler and cleaner.